OracleGoldenGate/21c/bin/deployment-main.sh

```diff
@@ -121,6 +126,53 @@ function setup_deployment_directories() {
     chown -R ogg:ogg {} \;
 }
 
+##
+##  r u n _ u s e r _ s c r i p t s
+##
+## Hook for launching custom scripts in the container before and after ogg start
+##     Default Values:
+##       - ${SETUP_USER_SCRIPTS}          : "${OGG_HOME}/scripts/setup"
+##       - ${STARTUP_USER_SCRIPTS}        : "${OGG_HOME/scripts/startup}"
+##
+## Scripts are run lexicographically and recursively from the directories pointed to by:
+##      - ${SETUP_USER_SCRIPTS} are executed prior to any other steps in the boot sequence
+##      - ${STARTUP_USER_SCRIPTS} are executed after ogg/nginx startup
+##
+function run_user_scripts {
+
+    local SCRIPTS_ROOT="${1}";
+
+    [ -z "$SCRIPTS_ROOT" ] && {
```

Comment:
`${SCRIPTS_ROOT}` will never be empty

Reply:
It won't be assuming that the function `run_user_scripts` is invoked with a non-empty value. Additional check should be OK i guess.